### ISSUE TITLE
Why does two-way sync favour Outlook over Google when I change an event time?

### ISSUE THREAD

**Issue by ravenmaka** [phw198/OutlookGoogleCalendarSync#1874]
**Are you already using OGCS?**
OGCS - v2.10.0.0 ‘Frequent User’
Microsoft Office Professional Plus 2019
Windows 10 Pro x64 22H2
Android v14, One UI v6.1

**Describe the question you have**
A clear and concise description of your question and what you would like clarifying.
I’m getting a little frustrated here, when I edit the finish time of an event in Google Calendar on my mobile device I get home and use OGCS to sync up with my Outlook calendar.  O<->G.  However, the Outlook event’s previous finish time either overwrites the new edited Google event finish time or it just completely ignores that there has been an edit at all.  Does two-way sync prioritise Outlook events over Google events, and if so why can’t I have it the other way around?  I don’t want to have to keep double checking event time changes following a successful sync.

Help much appreciated.  Thank you!

**Comment by phw198** [comment]
The calendar with the most recently modified item is the one synced to the other - so should be Google to Outlook with what you have described.

If you check the Google web version on https://calendar.google.com can you see the edited end time before OGCS syncs? Are there any other attributes being updated from Outlook when it syncs?

**Comment by ravenmaka** [comment]
I've found out today that it's a problem with the Android Google Calendar not saving/refreshing (I had to do it manually).  After pressed 'Refresh' in the app menu OGCS successfully passed the modified event time information over to Outlook as I'd hoped.

Unfortunately, there isn’t an option to set the Google Calendar sync interval to a smaller number on my phone so I had to do it manually.

Cheers

**Comment by ravenmaka** [comment]
I think I’ve sussed the issue – Google Calendar hadn’t automatically synced/saved, I hit ‘Refresh’ in the Android app last night then tried another OGCS sync on my PC this morning and it behaved as normal.  Google successfully passed the modified event time information over to Outlook as I hoped.

 

Unfortunately, there isn’t an option to set the Google Calendar sync interval to a smaller number but I’ll update my question with this info so that others experiencing this issue understand it’s the app not updating frequently so I had to do it manually.

 

Thank you for the quick response and keep up the excellent work [fantastic app I have already donated]

Kind regards,

Ian | ravenmaka
